CONTRIBUTING.md

```diff
@@ -95,28 +95,64 @@ yarn dev --force
 
 > You can now hack in the cloned repository of `api-platform-admin`.
 
-#### Running Admin through Storybook
+#### Running Admin Through Storybook
 
-If you don't have an existing API Platform application, or don't want to use `yarn link`, you can use one of the bundled example APIs, and visualize the admin through [Storybook](https://storybook.js.org/).
+If you do not have an existing project, you can contribute by visualizing directly the changes you made the sources through [Storybook](https://storybook.js.org/). 
+
+This development stack consists of two Docker containers: 
+- `pwa`: containing the `<Admin>` sources and Storybook;
+- `php`: holding the API sources.
+
+Additionally, this method allows testing the integration between Api-platform and the `admin` component by writing stories, scenarios and tests.
 
 Install everything:
 
 ```shell
 docker compose up
 ```
 
-Go to http://localhost, accept the self-signed certificate, and see the running Admin.
+Before accessing the Storybook instance, make sure to go to https://localhost to accept the self-signed certificate. Once it's done, you'll see the API documentation running on a customized version of Swagger UI.
 
-To stop and prune the simple API Platform backend:
+Now you can go to http://localhost:3000/ to see the Storybook instance in action. The changes you'll make in the source code will be hot-reloaded.
+
+To access run a command directly inside a container, run:
+```shell
+# Run a command in the php container 
+docker compose exec -T php your-command
+
+# Run a command in the pwa container
+docker compose exec -T pwa your-command
+```
 
 ### Testing Your Changes
 
-Before sending a Pull Request, make sure the tests pass correctly:
+Before sending a Pull Request, make sure the tests pass correctly.
+
+#### With Linked Sources
+
+If you have chosen to link `admin` sources, simply run:
```

Comment:
If my understanding is correct, `yarn test` should pass regardless of whether you have linked the sources or not. Same applies for Storybook tests I believe.